platform/core/src/extensions/ExtensionManager.ts

```diff
@@ -283,6 +283,11 @@ export default class ExtensionManager {
             // Default for most extension points,
             // Just adds each entry ready for consumption by mode.
             extensionModule.forEach(element => {
+              if (!element.name) {
```

Comment:
Not defining the name causes loading issues in various places, so it is worth having a check here on load.